### PR TITLE
Pydr 110 add string commands to redis interface

### PR DESCRIPTION
**Pull request by dlozina-macrometa**
## Description

- Added new Redis commands
- Added end-to-end-tests for commands
- Corrected SQL tests

Fixes # (issue)

## Type of change

- [X] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

In the code base, we have end-to-end tests

## Reviews

Please identify two developers to review this change

- [ ] @koshy-thomas 
- [ ] @edgargarciamacrometa 

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented on my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [X] I have added new functional tests that prove my fix is effective or that my feature works
- [X] Existing and new functional tests pass with my changes
- [X] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: c8/client.py [hunk 1]
```diff
@@ -2806,6 +2825,35 @@ def get_list_of_analyzer(self):
         """
         return self._search.get_list_of_analyzer()
 
+    def create_analyzer(self, name, analyzer_type, features=[], properties={}):
```
Comment (koshy-thomas) [inline c8/client.py:2828]: This needs to be removed

File: c8/client.py [hunk 2]
```diff
@@ -2806,6 +2825,35 @@ def get_list_of_analyzer(self):
...
+        :param features: The set of features to set on the analyzer generated fields.
+                         The default value is an empty array.
+        :type features: list
+        :returns: Returns analyzer object if analyzer created successfully
+        :rtype: dict
+        """
+        return self._search.create_analyzer(name, analyzer_type, features=features,
+                                            properties=properties)
+
+    def delete_analyzer(self, name):
```
Comment (koshy-thomas) [inline c8/client.py:2847]: This needs to be removed

File: c8/redis/core.py [hunk 3]
```diff
@@ -12,12 +12,6 @@ def build_request(collection, data):
     return request
 
 
-def response_handler(request, response):
-    if not response.is_success:
-        raise RedisServerError(response, request)
-    return response.body
-
-
 class RedisServerError(C8ServerError):
```
Comment (koshy-thomas) [inline c8/redis/core.py:15]: A question here should the RedisServerError be present in redis/core file or should it be present in exceptions.py file?

File: c8/client.py (hunk 1 above)
Comment (dlozina-macrometa) [reply c8/client.py:2828]: 👍

File: c8/client.py (hunk 2 above)
Comment (dlozina-macrometa) [reply c8/client.py:2847]: 👍

File: c8/redis/core.py (hunk 3 above)
Comment (dlozina-macrometa) [reply c8/redis/core.py:15]: That `exceptions.py` is full of one-liners that inherit the same classes (C8ClientError or C8ServerError). I would like to separate and refactor that file. This is a start Redis package has Redis Interface, core, and utils. The same would be excellent for queries, streams, workers, collections etc.

File: tests/e2e/test_redis.py [hunk 4]
```diff
@@ -8,103 +8,642 @@
 API_KEY
 FABRIC
```
Comment (koshy-thomas) [inline tests/e2e/test_redis.py:9]: We can remove the above lines

**Review by koshy-thomas — APPROVED** [review]

File: tests/e2e/test_redis.py (hunk 4 above)
Comment (dlozina-macrometa) [reply tests/e2e/test_redis.py]: Sure! That makes sense 👍

**Pull request merged**